architecture/environments/prod/main.tf

```diff
@@ -21,6 +21,15 @@ module "zones" {
   root_zone_url  = local.website_url
 }
 
+# Documentation
+resource "google_dns_record_set" "documentation" {
+  name         = "docs.${module.zones.root_zone.dns_name}"
+  managed_zone = module.zones.root_zone.name
+  type         = "CNAME"
+  ttl          = 300
+  rrdatas      = ["elpiscloud.readthedocs.io."]
```

Comment:
I am not sure about this but is there supposed to be a '.' at the end as well? Or just elpiscloud.readthedocs.io?

Reply:
Also, this is just me not understanding all this very well but you have set it up such that elpiscloud.readthedocs.io maps to docs.elpis.cloud - but then where is docs.elpis.cloud given an IP address?

Reply:
All of the examples here: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/dns_record_set have the trailing period. I guess we'll find out on plan/apply.

Reply:
> where is docs.elpis.cloud given an IP address

This entry is just adding a DNS record, the actual IP allocated where the e.g. EC2 instances are created and resolved (I'm assuming) by reference in this line `name = docs.${module.zones.root_zone.dns_name}`.

Reply:
It's also a CNAME record, which means it's just pointing at a URL, not an IP address.

Reply:
The dot indicates that the domain name is complete. A domain without a dot may get the resolver's default domain attached to it.